modules/govuk/manifests/apps/email_alert_api/checks.pp

```diff
@@ -6,6 +6,28 @@
   $ensure = present,
 ) {
 
+  sidekiq_queue_check {
+    'delivery_immediate_high':
+      latency_warning  => '2.5',
+      latency_critical => '5';
+    'delivery_immediate':
+      latency_warning  => '2.5',
+      latency_critical => '5';
+    'delivery_digest':
+      latency_warning  => '90',
+      latency_critical => '60';
+    'email_generation_immediate':
+    ;
+    'process_and_generate_emails':
+    ;
+    'default':
+    ;
+    'email_generation_digest':
+    ;
+    'cleanup':
+    ;
+  }
```

Comment:
This is looking really nice :+1: 

One thing I spotted when looking in to this Puppet resource declaration stuff is that you can have default values specified here [1].

1: https://puppet.com/docs/puppet/latest/lang_resources.html#resource-declaration-default-attributes

So, it might be possible to move the defaults from the `sidekiq_queue_check`, and make the latency parameters mandatory, and just specify all the values here, which might improve readability.

I think this is fine as well though, so don't feel like you have to change it.

Reply:
Thanks @cbaines , this does look much easier to read so I'll use this approach 😄

Reply:
Great :)